content/microservices/custom-transport.md

```diff
@@ -103,6 +103,22 @@ Hello world!
 
 Which means that our method handler was properly executed.
 
+When using a `CustomTransportStrategy` with [Interceptors](/interceptors) the handlers are wrapped into RxJS streams. This means that you need to subscribe to them in order to excute the streams underlying logic (e.g. continue into the controller logic after an interceptor has been excuted).
+
+An example of this can be seen below:
+
+```typescript
+async listen(callback: () => void) {
+  const echoHandler = this.messageHandlers.get('echo');
+  const streamOrResult = await echoHandler('Hello World');
+  if (isObservable(streamOrResult)) {
+    streamOrResult.subscribe();
+  }
+  console.log(streamOrResult);
```

Comment:
This is somewhat confusing. Why are we logging the `streamOrResult` here? In case of observables, it won't equal to the emitted value

Reply:
Did this along time ago but I think my logic was the same docs above for use without interceptors was doing this. Happy to omit the console.log though since you're write it does add anything